src/lib/bootloader/auto_client.rb

```diff
@@ -27,7 +29,18 @@ def run
     end
 
     def import(data)
-      Yast::Bootloader.Import(data)
+      begin
+        Yast::Bootloader.Import(data)
+      rescue Bootloader::UnsupportedBootloader => e
+        possible_values = BootloaderFactory.supported_names + ["default"]
+        AutoInstall.issues_list.add(:invalid_value, "bootloader", "loader_type",
+            e.bootloader_name,
+            _("Bootloader is not supported on this architecture. Possible values: ") + 
```

Comment:
`Bootloader is not supported on this architecture` it a generic message saying that bootloaders do not work here. That's not true.

Proposals:

- `The selected bootloader is not supported on this architecture.`
- Or if you know what was in the profile you could include is as well:
  `The selected bootloader ('%{bootloader}') is not supported on this architecture.`